wicamo96/FieldScout
# Dashboard

# Description

Added graphs to the dashboard to give you a quick look at population trends for each pest over the last month. 

## Type of change


# Testing Instructions

### Create database and populate data if necessary
- If you haven't created the FieldScout database, please do so in visual studio community.
- Use this to fill out tables and columns: [SQL data](https://github.com/wicamo96/FieldScout/blob/main/SQL/01_FieldScout_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/wicamo96/FieldScout/blob/main/SQL/02_FieldScout_Seed_Data.sql)

### Testing Greenhouse Management CRUD
- Click the execute with debugger solid green button to launch the backend in visual studio community.
- From the client directory in a git terminal, run `npm run dev`.
- Once the Vite + React window pops up, you should be directed to the log in page.
- Log in with the default email.
- Once logged in, you will be taken to the dashboard page which populates a graph for each pest
- Select a house from the dropdown at the top to view data from the last month for each pest filtered by that house

# Checklist:
